tests/regression/roles/sonic_dhcp_relay/defaults/main.yml

```diff
@@ -0,0 +1,231 @@
+            - address: 100::2
+            - address: 100::3
+          vrf_name: '{{ vrf2 }}'
+          vrf_select: true
+          source_interface: '{{ vlan2 }}'
+          max_hop_count: 8
+
+  - name: test_case_02
+    description: Update DHCP and DHCPv6 relay configuration
+    state: merged
+    input:
+      - name: '{{ interface1 }}'
+        ipv4:
+          server_addresses:
+            - address: 100.1.1.4
+          circuit_id: '%h:%p'
+          policy_action: 'discard'
+          vrf_select: false
+        ipv6:
+          vrf_select: false
+      - name: '{{ interface2 }}'
+        ipv4:
+          server_addresses:
+            - address: 101.1.1.2
+          circuit_id: '%h:%p'
+          max_hop_count: 8
+        ipv6:
+          server_addresses:
+            - address: 101::2
+            - address: 101::3
+          max_hop_count: 8
+
+  - name: test_case_03
+    description: Update DHCP and DHCPv6 relay configuration
+    state: merged
+    input:
+      - name: '{{ po1 }}'
+        ipv4:
+          server_addresses:
+            - address: 120.1.1.2
+            - address: 120.1.1.3
+          source_interface: '{{ vlan2 }}'
+          link_select: false
+          circuit_id: '%p'
+          max_hop_count: 8
+      - name: '{{ po2 }}'
+        ipv6:
+          server_addresses:
+            - address: 121::2
+            - address: 121::3
+          source_interface: '{{ vlan2 }}'
+          max_hop_count: 8
+
+  - name: test_case_04
+    description: Update DHCP and DHCPv6 relay configuration
+    state: merged
+    input:
+      - name: '{{ po1 }}'
+        ipv4:
+          link_select: true
+          circuit_id: '%i'
+          max_hop_count: 6
+        ipv6:
+          server_addresses:
+            - address: 120::2
+      - name: '{{ po2 }}'
+        ipv4:
+          server_addresses:
+            - address: 121.1.1.2
+      - name: '{{ vlan1 }}'
+        ipv4:
+          server_addresses:
+            - address: 110.1.1.2
+            - address: 110.1.1.3
+            - address: 110.1.1.4
+            - address: 110.1.1.5
+          circuit_id: '%i'
+          policy_action: 'append'
+
+  - name: test_case_05
+    description: Delete specific DHCP and DHCPv6 relay configurations
+    state: deleted
+    input:
+      - name: '{{ interface1 }}'
+        ipv4:
+          server_addresses:
+            - address: 100.1.1.2
+          source_interface: '{{ vlan2 }}'
+          link_select: true
+        ipv6:
+          server_addresses:
+            - address: 100::2
+          source_interface: '{{ vlan2 }}'
+      - name: '{{ interface2 }}'
+        ipv4:
+          circuit_id: '%h:%p'
+          max_hop_count: 8
+        ipv6:
+          server_addresses:
+            - address: 101::3
+          max_hop_count: 8
+      - name: '{{ vlan1 }}'
+        ipv4:
+          policy_action: 'append'
+
+  - name: test_case_06
+    description: Delete specific DHCP and DHCPv6 relay configurations
```

Comment:
Please describe this test case more specifically. (I assume it is intended to verify that deletion of all server addresses results in deletion of the other parameters for the affected interface/AF as well.)

Reply:
Have updated the description for the test cases 6, 7, and 8 achieving similar functionality with different approaches

Reply:
The updates look good.

Thanks.